README.md

```diff
@@ -2,10 +2,16 @@
 
 ## Requirements
 
-The following are installed.
+The followings must be installed and configured in advance to build and run Personium Unit offered from this repository.
 
-* docker
-* docker-compose
+* [Docker](https://docs.docker.com/get-docker/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Windows 10 User  
+Special requirements if you want to run Docker in Windows 10. Follow the [official instructions from Microsoft](https://docs.microsoft.com/en-us/windows/wsl/install-win10) to install WSL 2.  
+> I highly recommend to check out [A Linux Dev Environment on Windows with WSL 2, Docker Desktop and More](https://nickjanetakis.com/blog/a-linux-dev-environment-on-windows-with-wsl-2-docker-desktop-and-more) before you start installing anything.  
```

Comment:
It looks nice for person who want to know what WSL2 makes available to use.
But it seems to contains stuffs not related to developing personium, so I don't think it is `highly recommended` for personium beginners 🤔